network/Cargo.toml

```diff
@@ -38,6 +38,7 @@ ckb-build-info = {path = "../util/build-info"}
 num_cpus = "1.10"
 snap = "0.2"
 ckb-protocol = { path = "../protocol" }
+ipnetwork = "0.14"
```

Comment:
Why we need this external dependency? Do we need any feature from this crate?

Reply:
we need this crate to support IP CIDR.

Reply:
Why and when a node needs to ban IP range?

Reply:
user can ban IP range through rpc, reference to bitcoin:
https://bitcoincore.org/en/doc/0.18.0/rpc/network/setban/